src/components/page/blocks/BlocksColumn.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Blocks from './index.jsx';
+
+function BlocksColumn({
+  getCol,
+  blocksGroups,
+  defaultLayout,
+  blockShared,
+  blockRows,
+}) {
+  const getLayout = layout => {
```

Comment:
`getLayout` method extracted from `Blocks` to determine a content block's position using the `defaultLayout` as a fallback